src/Dictionaries/ExecutableDictionarySource.cpp

```diff
@@ -360,7 +275,7 @@ void registerDictionarySourceExecutable(DictionarySourceFactory & factory)
         /// It's OK for dictionaries created by administrator from xml-file, but
         /// maybe dangerous for dictionaries created from DDL-queries.
         if (check_config)
-            throw Exception("Dictionaries with Executable dictionary source is not allowed", ErrorCodes::DICTIONARY_ACCESS_DENIED);
+            throw Exception("Dictionaries with executable dictionary source is not allowed created from DDL are not allowed", ErrorCodes::DICTIONARY_ACCESS_DENIED);
```

Comment:
```suggestion
            throw Exception("Dictionaries with executable dictionary source from DDL are not allowed", ErrorCodes::DICTIONARY_ACCESS_DENIED);
```

Reply:
But why it is dangerous?

Reply:
Because non administrator can create them and they will have the same privileges as ClickHouse executable.